src/screens/HomeScreen.tsx

```diff
@@ -95,7 +95,9 @@ const HomeScreen = () => {
 						text="Delete Authenticators"
```

Comment:
Can you please localise this? It can be a leftover...

Reply:
Done.